### ISSUE TITLE
Parser for rebar.config.script does not recognise previously defined variables

### ISSUE THREAD

**Issue by xurxodiz** [inaka/elvis#577]
Running elvis on a rebar.config.script such as:

```
Vsn = 3.
[{minimum_otp_vsn, Vsn}].
```

throws an `unbound_var` exception on `Vsn`. Changing the period to a comma works around it, though for long files with many preparations it can be clunky to chain so many sentences that could be kept separate (rebar allows for this).

Nevertheless, this also happens for the explicitly available variables `CONFIG` and `SCRIPT` provided by rebar3 itself. They are also cause of an `unbound_var` error.

**Comment by paulo-ferraz-oliveira** [comment]
I don't know this syntax. How are you using it?

And how are you executing `rebar3` on top of this?

**Comment by paulo-ferraz-oliveira** [comment]
Correct, but I always thought the `rebar3` script syntax had to be somewhat like escript. The above has two `.`, for example.

**Comment by paulo-ferraz-oliveira** [comment]
In any case, sure, not something I think `elvis_core` is prepared for, but we might want to extend it, where this would be a feature, not a bug.
